olive/systems/docker/docker_system.py

```diff
@@ -217,3 +229,11 @@ def _run_container(
     def remove(self):
         self.docker_client.images.remove(self.image.tags[0], force=True)
         logger.info(f"Image {self.image.tags[0]} removed successfully.")
+
+
+def _print_docker_logs(logs, level=logging.INFO):
```

Comment:
I doubt we should log every line in error level if there was a `BuildError`. Those build logs with `error` keyword will be noises when debugging. Can we log build log as usually but add another error log indicating the build failed?

Reply:
the level will be error in case of BuildError happen. You can see line 69

Reply:
Yeah, I mean every line of build logs will start with `[ERROR]` and this is not expected. If I got an error when running Olive and search `error` to debug in my logs, I will get all build logs.

Reply:
all the logs are output by one line logging